aspnetcore/host-and-deploy/linux-nginx.md

```diff
@@ -181,13 +221,25 @@ To directly run the app on the server:
 1. Navigate to the app's directory.
 1. Run the app: `dotnet <app_assembly.dll>`, where `app_assembly.dll` is the assembly file name of the app.
 
+# [Ubuntu](#tab/linux-ubuntu)
+
 If the app runs on the server but fails to respond over the Internet, check the server's firewall and confirm port 80 is open. If using an Azure Ubuntu VM, add a Network Security Group (NSG) rule that enables inbound port 80 traffic. There's no need to enable an outbound port 80 rule, as the outbound traffic is automatically granted when the inbound rule is enabled.
 
+# [Red Hat Enterprise Linux](#tab/linux-rhel)
+
+If the app runs on the server but fails to respond over the Internet, check the server's firewall and confirm port 80 is open.
```

Comment:
I realize you're copy/pasting the original and tweaking it, but internet shouldn't be capitalized. There are a few more places that need this change, I didn't point them out. I wonder if Acrolinx was ever run on the original article?
```suggestion
If the app runs on the server but fails to respond over the internet, check the server's firewall and confirm port 80 is open.
```